scripts/release/main.go

```diff
@@ -385,7 +385,7 @@ func (r *releaseCommand) autoversionFile(ctx context.Context, file, channel, ver
 			// Utilize the index where the match was found to replace the correct part. The only
 			// match group is the version.
 			if match := matchRe.FindStringSubmatchIndex(line); match != nil {
-				logger.Info(ctx, "updating version number", "line_number", i+1, "match", match)
+				logger.Info(ctx, "updating version number", "line_number", i+1, "match", match, "old_version", line[match[2]:match[3]])
```

Comment:
Potential here for panic; can you check that `match` has the expected number of elements first?

Reply:
If the regex matches we can expect that `match` to contains all the elements, but I can add an extra check just to be safe.

Reply:
@dannykopping turned this into a more robust implementation in 26df0e514, wdyt?

Reply:
LGTM 👍